pacai/util/probability.py

```diff
@@ -50,8 +46,8 @@ def nSample(distribution, values, n):
     return samples
 
 def sample(distribution, values = None):
```

Comment:
Check size of `distribution` first.

Later on, `total = distribution[0]` would throw.